WordPress/src/test/java/org/wordpress/android/ui/uploads/UploadStarterConcurrentTest.kt

```diff
@@ -95,6 +95,7 @@ class UploadStarterConcurrentTest {
 
         fun createDraftPostModel() = PostModel().apply {
             status = PostStatus.DRAFT.toString()
+            setIsLocallyChanged(true)
```

Comment:
Out of context opinion:

I think there is a mismatch between what this method does and what how it's named. This was the case before this PR as well, since there is already a call that changes `dateLocallyChanged`. I'd personally do one of these 2:

1. Change the name of the method to `createLocallyChangedDraftPostModel`
2. Or, accept a `isLocallyChanged` parameter with a default value `false`

I think this issue is even more apparent in the `UploadStarterTest`. Not only does the method do things that are not clear from its name, it even does something completely opposite to it. The method name is `createDraftPostModel` whereas it has a parameter to change its status to something else. I think in this case, it'd have been better to call it `createPostModel` instead and accept a couple more parameters for `id`, `title` and `isLocallyChanged`.

---

I am totally fine with this change in this PR, but I hoped that it'd still be valuable to share my thoughts on the subject for future.

Reply:
Thanks for sharing;)! 

I completely agree that the `createDraftPostModel` name especially the one in the `UploadStarterTest` is quite confusing - since it accepts `PostStatus` parameter (fixed in 1e32aa9322426be2a60b0f9e3306adb15c7234d7). 

However, for the other attributes I feel like it's a method in test Fixtures not a regular method. I think the fixtures methods should accept only attributes which are necessary for testing different scenarios and they should setup the models/method results to the most convenient values for the unit tests (eg `isPublishable` returns true since we don't need to test posts which are not publishable). 

We could make `id`, `title`, `isLocallyChanged` optional parameters and never override them, but I think it's kind of nice to see from a first glance that we don't need to change them in any of the unit tests. If we made them as mandatory parameters, it'd make the tests much less clear imho - as right now you can see what the tests needs to change in order to test a specific scenario.

It's a matter of personal opinion I guess 🤷‍♂ . Just to be clear, if it was a regular method in a production code, I'd completely agree with you.

Reply:
> but I think it's kind of nice to see from a first glance that we don't need to change them in any of the unit tests

That makes a lot of sense!

> Just to be clear, if it was a regular method in a production code, I'd completely agree with you.

I was thinking more about that then tests and I like the trade-off you make for tests. Thanks for entertaining this discussion :)